bulk-print-fail-action-group.tf

```diff
@@ -0,0 +1,11 @@
+module "cmc-bulk-print-fail-action-group" {
+  source = "git@github.com:hmcts/cnp-module-action-group"
+  location = "${var.location}"
+  env = "${var.env}"
+
+  resourcegroup_name = "${azurerm_resource_group.rg.name}"
+  action_group_name = "Bulk Print Fail Alert - ${var.env}"
+  short_name = "BPF_alert"
+  email_receiver_name = "Bulk Print Alerts"
```

Comment:
Are short_name and email_receiver_name getting used? Custom email subject can be in this file. Values look bit scattered to me?

Reply:
All values of this file can be inside be cmc-bulk-print-fail-alert.tf if needed. With unused values removed, all can be in one place I reckon.

Reply:
Yeah they're all used. The custom email subject doesn't apply to the action group - it's a property of the alert that consumes the action group.